src/main/frontend/modules/outliner/core.cljs

```diff
@@ -600,7 +601,9 @@
       (let [root-page (:db/id (:block/page (:data root)))
             target-page (:db/id (:block/page (:data target-node)))
             not-same-page? (not= root-page target-page)
-            opts (cond-> {:outliner-op :move-subtree}
+            opts (cond-> {:outliner-op :move-subtree
+                          :move-blocks [(:db/id (get-data root))]
+                          :target (:db/id (get-data target-node))}
```

Comment:
Both `move-blocks` and `target` are added so that we don't need to go through the whole blocks structure when re-rendering, check `frontend.db.model/get-start-id-for-pagination-query`.